public/css/list/redundancy-group-list-item.less

```diff
@@ -0,0 +1,21 @@
+.redundancy-group-list-item.list-item {
```

Comment:
Is this a specificity thing or why is `.list-item` added here?

Reply:
Yes, I wanted to make the rules more specific. But now that you point it out, this seems unnecessary.